pages/common/enable.md

```diff
@@ -10,3 +10,7 @@
 - Disable a builtin (works in Bash only):
 
 `enable -n {{command}}`
+
+- Re-enable a builtin:
+
+`enable {{command}}`
```

Comment:
From a reading point of view, shouldn't enabling have to be shown before disabling?